src/midi-router.js

```diff
@@ -9,7 +9,7 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking");
+    flock = fluid.require("flocking"); // eslint-disable-line no-unused-vars
```

Comment:
In other words, if a global variable is only used by IoC, linting has no way of knowing about it.